sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/implementation/package-info.java

```diff
@@ -4,6 +4,6 @@
 
 /**
  * Package containing the implementations and inner classes for PhoneNumberAdminClient. The phone numbers client uses
- * Azure Communication Services to acquire and manage phone numbers.
```

Comment:
nit. The class name is PhoneNumberAdminClient but the comment says The phone number**s** client